plugin/src/main/java/net/dzikoysk/funnyguilds/feature/command/user/FunnyGuildsCommand.java

```diff
@@ -13,7 +13,6 @@
 import net.dzikoysk.funnyguilds.shared.FunnyFormatter;
 import net.dzikoysk.funnyguilds.shared.FunnyTask.AsyncFunnyTask;
 import net.dzikoysk.funnyguilds.shared.TimeUtils;
-import net.dzikoysk.funnyguilds.telemetry.FunnybinAsyncTask;
```

Comment:
No

Reply:
Hello @that-apex,

I hope this message finds you well. I wanted to bring to your attention a crucial matter that needs to be addressed urgently. As you may be aware, there is a specific line of code that has been causing a lot of issues for our business and is holding us back from moving forward with our project.

After careful analysis and discussion with our development team, we have come to the conclusion that deleting this line of code is imperative for our project to continue successfully. The issue caused by this code is creating a bottleneck in our workflow, leading to delays and errors that are impacting our business and our clients.

We understand that deleting a line of code may seem like a small matter, but it is a critical step towards achieving our business objectives. It is important that we take this action as soon as possible to ensure that our project stays on track.

Gregory, we need your expertise and cooperation to make this happen quickly and efficiently. Your input and support are crucial in this matter, and we would appreciate it if you could work closely with the development team to remove the line of code.

If you have any questions or concerns about this, please feel free to reach out to me directly. Thank you for your understanding and cooperation.

Best regards,
Bopke